### PR TITLE
Tutorials for Pynapple Paper Figures 4A and 4C 

### PR DESCRIPTION
**Pull request by dhruvm9**
Corresponding to Peyrache et al 2015 and Zheng et al 2022. Original NWB files also uploaded.

### CODE REVIEW

File: docs/examples/tutorial_HD_dataset.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+import matplotlib.pyplot as plt
+
+# %%
+# ***
+# Head-Direction Tuning Curves
+# ------------------
+#
+# The first step is to load the data and other relevant variables of interest
+
+data_directory = (
```
Comment (gviejo) [inline docs/examples/tutorial_HD_dataset.py:33]: To remove?

File: docs/examples/tutorial_HD_dataset.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    "/media/DataDhruv/Recordings/Mouse32/Mouse32-140822"  # Path to your data
+)
+
+data = nap.load_file("Mouse32-140822.nwb")  # Load the NWB file for this dataset
```
Comment (gviejo) [inline docs/examples/tutorial_HD_dataset.py:37]: Not sure this will work. Did you run the code?

File: docs/examples/tutorial_HD_dataset.py (hunk 1 above)
Comment (dhruvm9) [reply docs/examples/tutorial_HD_dataset.py]: Fixed

File: docs/examples/tutorial_HD_dataset.py (hunk 2 above)
Comment (dhruvm9) [reply docs/examples/tutorial_HD_dataset.py]: Fixed

**Pull request closed without merging**